### PR TITLE
Feature/UI redesign footer #861

### LINKED ISSUE #861: Improve UI from the main page
Les't changes the footer, sidebar and header from the dashboard.

- [x] Footer
- [x] Header
- [x] Sidebard

### PR DESCRIPTION
**Pull request by fvives99**
### What does this PR do?

- Resolve #861 

### Steps to test

1. make start
2. navigate to http://localhost:3000/ and see footer&header

#### CheckList

- [ ] Follow proper Markdown format
- [ ] The content is adequate
- [ ] The content is available in both english and spanish
- [ ] I Ran a spell check

### CODE REVIEW

**Comment by fvives99** [comment]
See attached images:

**Comment by fvives99** [comment]
See images for colapsed sidebar

image
image

**Review by Torresmorah — APPROVED** [review]
LGTM

File: .env.jungle [hunk 1]
```diff
@@ -62,6 +62,7 @@ REACT_APP_NAME=$npm_package_name
 REACT_APP_TITLE=EOS Jungle Testnet Network Dashboard
 REACT_APP_DEFAULT_PRODUCER_LOGO=https://eosio.online/images/jungle.jpg
 REACT_APP_FOOTER_LINKS=[{"text":"Website","src":"https://jungletestnet.io/"},{"text":"Block Explorer","src":"https://jungle3.bloks.io"},{"text":"Developer Guidelines","src":"https://guide.eoscostarica.io"},{"text":"GitHub","src":"https://github.com/eoscostarica/eosio-dashboard"}]
+REACT_APP_FEATURE_BUG=[{ "text": "Feature Request/Bug Report", "src": "https://github.com/eoscostarica/eosio-dashboard/issues/new/choose"}]
```
Comment (xavier506) [inline .env.jungle:65]: I think this text reads better like this: 

`"text": "Report a Bug or Request a Feature"`

File: webapp/Dockerfile [hunk 2]
```diff
@@ -4,6 +4,7 @@ ARG port
 ARG react_app_title
 ARG react_app_default_producer_logo
 ARG react_app_footer_links
+ARG reac_app_privacy-terms
```
Comment (xavier506) [inline webapp/Dockerfile:7]: typo `reac_app..` missing a `t`

File: webapp/src/components/Footer/index.js [hunk 3]
```diff
@@ -6,21 +6,23 @@ import {
...
 const useStyles = makeStyles(styles)
+const preventDefault = (event) => event.preventDefault()
 
 const Footer = () => {
   const classes = useStyles()
 
   return (
     <div className={classes.wrapper}>
-      <Grid container item xs={12} sm={8}>
+      <Grid container item xs={12} sm={8} justifyContent='flex-start'>
```
Comment (xavier506) [inline webapp/src/components/Footer/index.js:25]: is there an opportunity to refactor from `grid->box` to `divs` here?

**Review by xavier506 — CHANGES_REQUESTED** [review]
@fvives99  please make sure design styles match @Edwinjrc 's designs

**Review by tetogomez — CHANGES_REQUESTED** [review]
Please Refactor this PR in order to Remove Box and Grid component

also, there are several commented code

File: webapp/src/components/Footer/index.js [hunk 4]
```diff
@@ -34,32 +28,54 @@ const Footer = () => {
...
+
+      <div className={classes.gridFooter}>
+          <div>
+            <Link
+              href="https://github.com/eoscostarica/eosio-dashboard"
+              onClick={preventDefault}
+              style={{ textDecoration: 'none' }}
+            >
+              <Typography className={classes.sidebarFooterText}>
+                This app is Open Source
```
Comment (xavier506) [inline webapp/src/components/Footer/index.js:41]: Why arent we using i18n?

**Review by tetogomez — APPROVED** [review]
LGTM

Just check the code factor comments

**Review by tetogomez — APPROVED** [review]

**Pull request merged**